pajbot/bot.py

```diff
@@ -88,6 +88,12 @@ def __init__(self, config, args):
 
         if config["main"].getboolean("verified", False):
             TMI.promote_to_verified()
+        elif config["main"].getboolean("known", False):
+            TMI.promote_to_known()
+
+        whisper_output = config["main"].getint("wisper_output", 1).lower()
+        if whisper_output in [0, 2]:
+            TMI.change_whisper_output(whisper_output)
```

Comment:
I don't like it being called "whisper output". Can we call it "whisper limiting" or "whisper rate-limiting" or something like that?